scala/integration/bsp/src/org/jetbrains/plugins/scala/bsp/config/ScalaPluginConstants.scala

```diff
@@ -0,0 +1,13 @@
+package org.jetbrains.plugins.scala.bsp.config
+
+import com.intellij.openapi.externalSystem.model.ProjectSystemId
+object ScalaPluginConstants {
+  val ID = "scalabsp"
+  val SYSTEM_ID = new ProjectSystemId(ID, "Scala BSP")
+  val WORKSPACE_FILE_NAMES: List[String] = List() // TODO: what are these? In bazel they have listOf("WORKSPACE", "WORKSPACE.bazel", "MODULE.bazel", "WORKSPACE.bzlmod")
```

Comment:
Remove this TODO, will be resolved at the meeting.
Side note: this is already implemented, please rebase